app/scripts/modules/core/src/managed/managedResourceStatusConfig.tsx

```diff
@@ -37,11 +37,10 @@ export const viewConfigurationByStatus: { [status in ManagedResourceStatus]: IVi
     popoverContents: (resourceSummary: IManagedResourceSummary) => (
       <>
         <p>
-          <b>Action is being taken to resolve a drift from the declarative configuration.</b>
+          <b>Spinnaker is taking action to resolve a drift from this resource's desired state.</b>
```

Comment:
This is strictly better than what was there before, but I think people still find the "resolve a drift" language confusing. I don't think people think of a new version becoming available as a resource drifting away from desired state.

Maybe "difference" would be less confusing than "drift"?

Reply:
+1 for `difference`

Reply:
I think you've mostly convinced me, will swap it out. As a side note on your overall comment about deployments, I personally believe that almost _any_ way of communicating deployments or new versions as a change in actual vs. desired state is pretty academic for most people. My overall strategy so far has been to focus on making the environments experience really great at describing the deployment/version view of the world, and keep resource statuses more focused on state/actuation. I think there's probably room in the future to think about creating a dedicated resource status for 'there's a new/different version that needs to be deployed', but unclear how valuable it would be.